packages/seed/src/cli.ts

```diff
@@ -197,6 +199,81 @@ function addRunCommand(cli: Argv) {
     );
 }
 
+function addPublishCommands(cli: Argv) {
+    cli.command("publish", "Publish releases", (yargs) => {
+        yargs
+            // TODO: Implement CLI releasing (currently npm publishing)
+            .command(
+                "generator <generator>",
+                "Publishes all latest versions of the generators to DockerHub unless otherwise specified. To filter to certain generators pass in the generator IDs as a positional, space delimited list.",
+                (yargs) =>
+                    yargs
+                        .positional("generator", {
+                            type: "string",
+                            demandOption: true,
+                            description: "Generator(s) to register"
+                        })
+                        // Version is a reserved option in yargs...
+                        .option("ver", {
+                            type: "string",
+                            demandOption: false
+                        })
+                        .option("versions-file", {
+                            type: "string",
+                            demandOption: false,
```

Comment:
should this eventually be true? is the primary thing blocking migrating all of the changelog.mds to the new format